plugins/device/deviceutils/store-utils.go

```diff
@@ -51,14 +52,43 @@ func GetDevicesFromPeer(peerInfo *peer.Peer) ([]deviceapi.Info, error) {
 	return deviceInfo, nil
 }
 
-// DeviceInList checks whether the given device is in list of devices or not.
-func DeviceInList(reqDevice string, devices []deviceapi.Info) bool {
-	for _, key := range devices {
+// SetDeviceState sets device state and updates device state in etcd
+func SetDeviceState(peerID, deviceName, deviceState string) error {
+
```

Comment:
I just noticed that your line breaks are placed randomly through-out your PRs. There are extra lines where they shouldn't be and there are no line breaks where they should be.